### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
In this project, I build a simple HTML list of To Do tasks.

## Type of change

- [ ] Make sure that there are no linter errors
- [ ] Make sure that you used correct GitHub Flow
- [ ] Make sure that you documented your work in a professional way
- [ ] On page load render the dynamically created list of tasks.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  mode: 'development',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+};
```
Comment (descholar-ceo) [inline webpack.config.js]: - [ ] It was required to have an HTML file configured in the Webpack config settings as a template using the HTML Webpack Plugin.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            type="checkbox"
+            aria-label="..."
+          />
+          <span>${task.description}</span>
+        </div>
+        <i class="btn btn-outline-primary fs-3 bi bi-three-dots-vertical"></i>
+    `;
+  taskList.innerHTML = taskListContent;
+  tasksList.appendChild(taskList);
+});
```
Comment (descholar-ceo) [inline src/index.js]: - [ ] We should have this code wrapped inside the function, this is how the requirement says
Screenshot from 2022-07-06 00-39-36

**Review by descholar-ceo — APPROVED** [review]
.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @Owusu-Desmond , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
👍🏽 Implemented a half of the project's requirements ✔️ 
👍🏽 PR has descriptive title and description ✔️ 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RolandM99 — COMMENTED** [review]
Hi @Owusu-Desmond , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌

You have submitted a project, that failed some linters check. Linter result should be green, this means that your last commit should have a green check mark (✅). Kindly, fix the linter errors and then ask for a re-review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Owusu-Desmond 

Your project is complete! Great job for making the previous reviewers requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- Validations correctly implemented✔️ 
- No linter errors✔️ 
- captured video design well followed ✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**